source/common/config/rds_json.cc

```diff
@@ -275,15 +275,22 @@ void RdsJson::translateRoute(const Json::Object& json_route, envoy::api::v2::rou
 
     JSON_UTIL_SET_STRING(json_route, *action, prefix_rewrite);
 
+    int host_rewrite_options = 0;
```

Comment:
Please revert this file. We no longer add new features to the v1 API.

Reply:
File reverted.